src/main/scala/it/pagopa/interop/backendforfrontend/error/BFFErrors.scala

```diff
@@ -56,5 +56,10 @@ object BFFErrors {
   final case class EServiceNotFound(eServiceId: UUID) extends ComponentError("0013", s"EService $eServiceId not found")
 
   final case class TenantNotFound(tenantId: UUID) extends ComponentError("0014", s"Tenant $tenantId not found")
+  final case class InvalidInterfaceFileDetected(eServiceId: String, contentType: String, technology: String)
```

Comment:
this exception must be added to the `handleError` function, otherwise the response would be a 500 instead of a 400.
Code example to add
```
case Failure(err: InvalidInterfaceFileDetected)   => badRequest(err, logMessage)
```